app/controllers/posts_controller.rb

```diff
@@ -22,6 +22,14 @@ def create
     end
   end
 
+  def destroy
```

Comment:
- [ ] Great job so far 👏🏻 But you need to add this `load_and_authorize_resource` to the top of the Post Controller to make sure your authorization rules are enforceable and use **CanCanCan** for post authorization 😉 👍🏻 
- [ ] Also consider adding this to the Comment Controller ✍🏻

- Note: you could also add `authorize_resource` if you had this line eg;
`Before_action:set_name, only: %[show update destroy] ` at the top of your controller